Azure-Samples/Cognitive-Services-Voice-Assistant
# Add support for linux service

## Purpose
Added script to install service on linux.
Updated readme for service on raspberry pi.
Moved scripts around.

Closes #233 

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
build the linux version
Follow the instructions in the RaspberryPi.md
reboot your linux device.

See that it is running
